lisa/exekall_customize.py

```diff
@@ -177,15 +178,21 @@ def _finalize_expr_val(self, expr_val, artifact_dir, testcase_artifact_dir):
 
     @classmethod
     def get_tag_list(cls, value):
+        tags = []
```

Comment:
Ditto about splitting changes.

Reply:
this one does belong to here, I am adding tags to a type I introduce here